src/App.js

```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './component/Calculator';
 
 function App() {
```

Comment:
- [ ] As required by the project requirements, I kindly suggest that you make `App.js` component class-based. In future Milestones, you'll convert it back to a function based component.

## EXAMPLE
```
class App extends React.Component {
  
}
```